en/flight_modes_rover/differential.md

```diff
@@ -0,0 +1,132 @@
+| [Stabilized](#stabilized-mode) | Directly map stick input to motor commands.                              | Stick input creates a yaw rate setpoint for the control system to regulate. If this setpoint is zero (stick is centered) the control system will maintain the current yaw (heading) of the rover. | Yaw rate and yaw.                               |
+| [Position](#position-mode)     | Stick input creates a speed setpoint for the control system to regulate. | Stick input creates a yaw rate setpoint for the control system to regulate. If this setpoint is zero (stick is centered) the control system will keep the rover driving in a straight line.       | Yaw rate, yaw, speed and global position (GPS). |
+
+### Manual Mode
+
+In this mode the stick inputs are directly mapped to motor commands. The rover does not attempt to maintain a specific orientation or compensate for external factors like slopes or uneven terrain!  
+The user is responsible for making the necessary adjustments to the stick inputs to keep the rover on the desired course.
+
+| Stick                  | Effect                              |
+| ---------------------- | ----------------------------------- |
+| Left stick up/down     | Drive the rover forwards/backwards. |
+| Left stick centered    | Zero forward speed input.           |
+| Right stick left/right | Yaw the rover to the left/right.    |
+| Right stick centered   | Zero yaw rate input                 |
+
+For the configuration/tuning of this mode see [Manual mode](../config_rover/differential.md#manual-mode).
+
+### Acro Mode
+
+::: info
+This mode requires a yaw rate measurement.
+:::
+
+In this mode the vehicle regulates its yaw rate to a setpoint (but does not stabilize heading or regulate speed).
+
+| Stick                  | Effect                                                                                      |
+| ---------------------- | ------------------------------------------------------------------------------------------- |
+| Left stick up/down     | Drive the rover forwards/backwards.                                                         |
+| Left stick centered    | Zero forward speed input.                                                                   |
+| Right stick left/right | Create a yaw rate setpoint for the control system to regulate.                              |
+| Right stick centered   | The control system will attempt to maintain a zero yaw rate (minimal disturbance rejection) |
+
+For the configuration/tuning of this mode see [Acro mode](../config_rover/differential.md#acro-mode).
+
+### Stabilized Mode
+
+::: info
+This mode requires a yaw rate and yaw estimate.
+:::
+
+In this mode the vehicle regulates its yaw rate to a setpoint and will maintain its heading if this setpoint is zero (but does not regulate speed).  
+Compared to [Acro mode](#acro-mode), this mode is much better at driving in a straight line as it can more effectively reject disturbances.
```

Comment:
Yes, just what we need. This is kind of the comparison I was hoping for in the table. Still thinking about that.